frontend/components/templates/Profile.tsx

```diff
@@ -0,0 +1,208 @@
+import { omit } from 'lodash'
+import ProfileCard, { type ProfileCardProps } from '../molecules/ProfileCard'
+import SummaryCard from '../molecules/SummaryCard'
+import { type ITag } from '../molecules/TagsInput'
+type MITag = ITag & { isWatched: boolean }
+// const tempTags: MITag[] = [
+//     {
+//         id: 1,
+//         name: 'Typescript',
+//         description: 'abcd',
+//         slug: 'typescript',
+//         isWatched: false,
+//     },
+//     {
+//         id: 2,
+//         name: 'Typescript',
+//         description: 'abcd',
+//         slug: 'typescript',
+//         isWatched: true,
+//     },
+//     {
+//         id: 3,
+//         name: 'Typescript',
+//         description: 'abcd',
+//         slug: 'typescript',
+//         isWatched: false,
+//     },
+//     {
+//         id: 4,
+//         name: 'Typescript',
+//         description: 'abcd',
+//         slug: 'typescript',
+//         isWatched: true,
+//     },
+// ]
+
+// const tempCardProps = {
+//     title: 'Question Title',
+//     content: 'Lorem ipsum dolor sit amet consectetur. Sed amet at id sit proin in.',
+//     tags: tempTags,
+//     date: '2022-12-02',
+//     isBookmarked: true,
+//     // metadata: {
+//     //     votes: 99,
+//     //     answers: 99,
+//     //     views: 99,
+//     //     author: 'Ian Urriza',
+//     // },
+//     vote_count: '10',
+//     slug: 'questions',
+// }
+
+type TSummaryCard = {
+    id: number
+    title?: string
+    content: string
+    tags: MITag[]
+    updated_at: string
+    vote_count: string
+    slug: string
+}
+
+type ProfileQuestionData = {
+    top_questions: TSummaryCard[]
+    top_answers: Array<TSummaryCard & { question: { slug: string } }>
+    bookmarked_questions: Array<{
+        bookmarkable: TSummaryCard
+    }>
+    bookmarked_answers: Array<{
+        bookmarkable: TSummaryCard & { question: { slug: string } }
+    }>
+}
+type ProfileData = ProfileCardProps & ProfileQuestionData
+
+type ProfileLayoutProps = {
+    data: ProfileData
+    toggleFollow?: (input: { variables: { userId: number } }) => void
+    isPublic: boolean
+}
+
+const ProfileLayout = ({ data, toggleFollow, isPublic }: ProfileLayoutProps): JSX.Element => {
+    const renderActivities = (): JSX.Element => {
+        return (
+            <div className="w-full space-y-4 bg-white pb-4 drop-shadow-md">
+                <div className="bg-[#F5D3CF]  p-2 font-semibold leading-6">ACTIVITY</div>
```

Comment:
Please set the colors in the tailwind config file and apply changes to color classes. 

* _You can copy from here for the mean time_: https://github.com/framgia/sph-sun-overflow/pull/266/files#diff-8fdab0c1827a182dfaa61b1253086037a44105350409f6701c49ebe130f6705b